src/client/src/containers/Details/index.tsx

```diff
@@ -169,16 +179,15 @@ const Details = ({
                       defaultMessage="Author:"
                     />
                   </div>
-                  <div className={grid.col8}>
-                    {<ReactMarkdown source={detailInfo.author} /> ||
-                      intl!.formatMessage(messages.none)}
-                  </div>
+                  {(detailInfo.author &&
+                    renderFormattedData(detailInfo.author, false, true)) ||
+                    intl!.formatMessage(messages.none)}
```

Comment:
jw, how is `intl!` different from `intl`?

Reply:
It just says that the property mentioned might or might not exist on `intl`